prov/verbs/src/verbs_msg_ep.c

```diff
@@ -377,6 +377,8 @@ int fi_ibv_open_ep(struct fid_domain *domain, struct fi_info *info,
 
 	ofi_atomic_initialize32(&ep->unsignaled_send_cnt, 0);
 	ofi_atomic_initialize32(&ep->comp_pending, 0);
+	ep->send_signal_thr = (ep->info->tx_attr->size * 4) / 5;
+	ep->send_comp_thr = (ep->info->tx_attr->size * 9) / 10;
```

Comment:
Patch looks fine to me, but while we're changing this, can we add a comment regarding why these values were chosen?

Reply:
The values were chosen such that a send is signalled and its completion is polled before we fill up the send queue.

Reply:
Yes, I agree that we need to provide some details.
Done